diff --git a/setup.py b/setup.py
index a7998c22f1..a3746e12b7 100644
--- a/setup.py
+++ b/setup.py
@@ -2,9 +2,9 @@
 
 dependencies = [
     "blspy==1.0.2",  # Signature library
-    "chiavdf==1.0.1",  # timelord and vdf verification
+    "chiavdf==1.0.2",  # timelord and vdf verification
     "chiabip158==1.0",  # bip158-style wallet filters
-    "chiapos==1.0.2",  # proof of space
+    "chiapos==1.0.3",  # proof of space
     "clvm==0.9.6",
     "clvm_rs==0.1.7",
     "clvm_tools==0.4.3",
